akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -68,6 +72,11 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
     "contain the class name where the first log was called" in {
       val eventFilter = EventFilter.custom({
         case l: LogEvent if l.logClass == classOf[ActorLoggingSpec] ⇒ true
+        case l: LogEvent if isScala211 ⇒
+          // TODO remove in Akka 2.6 when we drop Scala 2.11
+          // the class with 2.11 is like
+          // ActorLoggingSpec$$anonfun$1$$anonfun$apply$mcV$sp$26$$anonfun$apply$6$$anonfun$apply$7
+          l.logClass.getName.startsWith(classOf[ActorLoggingSpec].getName)
```

Comment:
'startsWith' or is a regex better or overkill

Reply:
good enough ™️